minijail/util.c

```diff
@@ -102,6 +102,18 @@ static void attribute_no_optimize alias(const void *var)
 	(void)var;
 }
 
+void print_log_level(int priority, int fd) {
+	if (priority == LOG_INFO) {
+		dprintf(fd, "INFO ");
+	} else if (priority == LOG_WARNING) {
+		dprintf(fd, "WARN ");
+	} else if (priority == LOG_ERR) {
+		dprintf(fd, "ERR ");
+	} else {
+		dprintf(fd, "DEBUG ");
+	}
+}
+
```

Comment:
In general looks good to me...we just need to be really careful to not touch the minijail code in a way that prevents us from updating it.